utils/package.json

```diff
@@ -5,23 +5,23 @@
   "author": "NewRelic",
   "license": "Apache-2.0",
   "scripts": {
-    "validate-quickstart-install-plans": "node validate-quickstart-install-plans.js",
-    "create-validate-install-plans": "node create-validate-install-plans.js",
-    "create-validate-pr-quickstarts": "node create_validate_pr_quickstarts.js",
+    "validate-quickstart-install-plans": "ts-node validate-quickstart-install-plans.js",
```

Comment:
these might have to be `npx ts-node`. depends on how these are installed i think.